### DISCUSSION TITLE
Color Option for UI buttons

### DISCUSSION THREAD

**Discussion by angelcox-fl** [IBM/mac-ibm-notifications#97] [General]
**Is your feature request related to a problem? Please describe.**
Blue is a boring color, I think that is a _important_ problem

**Describe the solution you'd like**
A color code option for inclusion in button UI elements to help create visibility, ideally pink

**Describe alternatives you've considered**
A limited set of color options

**Additional context**
![image](https://user-images.githubusercontent.com/88393538/176538826-4dda2a16-70e9-4ce0-a4eb-d347d14db0cd.png)

**Comment by angelcox-fl** [comment]
I will accept that the button cannot be scented

**Comment by angelcox-fl** [comment]
seriously though, the ability to add higher contract UI color for color blind/visually impaired individuals would be great for accessibility, ideally if the system UI highlight color could be loaded for button highlights - that would be more than enough, however overrides for color codes to allow for consistence with fall back to high contrast system colors would be the most ideal.

**Comment by SMartorelli** [comment]
Hi @angelcox-fl, IBM Notifier already support all the native macOS accessibility features and it act as any other native macOS interface so if the individual change for example the accent colour in system settings or enable the high contrast accessibility feature this reflects in the IBM Notifier interfaces as you can see from the below examples
<img width="641" alt="acc_high_contrast" src="https://user-images.githubusercontent.com/23209655/176630547-0f57331f-01be-4dd3-99aa-580c9b1b003c.png">
<img width="641" alt="red_accent_color" src="https://user-images.githubusercontent.com/23209655/176630557-1e4ce188-d898-4b59-a3b2-1620fea5d647.png">

**Comment by SMartorelli** [comment]
@angelcox-fl Did you had any chance to check if my answer was helpful? Tomorrow I will close the issue but I can move it to the Discussions section if you want to talk more about it

**Comment by SMartorelli** [comment]
Issue have been converted in a Discussion
